introduction_to_amazon_algorithms/jumpstart_text_generation/Amazon_JumpStart_Text_Generation.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        f"jumpstart-cache-prod-{aws_region}", "models_manifest.json", "models_manifest.json"
</pre><p>@julkroll do we need have these files in sagemaker-sample-files to unify our data sources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
These are not static sources and keeps getting updated as we our scripts/models etc.

Reply:
I agree with Vivek, JumpStart manages their own bucket/artifacts, so it's okay to use this bucket as a data source.